src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -1361,90 +1320,66 @@ public static void IndexOf_SingleLetter(string s, char target, int startIndex, i
             bool safeForCurrentCulture =
                 IsSafeForCurrentCultureComparisons(s)
                 && IsSafeForCurrentCultureComparisons(target.ToString());
-
-            ReadOnlySpan<char> span = s.AsSpan();
+            
             var charArray = new char[1];
-            charArray[0] = target;
-            ReadOnlySpan<char> targetSpan = charArray;
-
-            int expectedFromSpan = expected == -1 ? expected : expected - startIndex;
-
+            charArray[0] = target;            
+            
             if (count + startIndex == s.Length)
             {
                 if (startIndex == 0)
                 {
                     Assert.Equal(expected, s.IndexOf(target));
                     Assert.Equal(expected, s.IndexOf(target.ToString(), StringComparison.Ordinal));
                     Assert.Equal(expected, s.IndexOf(target.ToString(), StringComparison.OrdinalIgnoreCase));
-
-                    Assert.Equal(expectedFromSpan, span.IndexOf(targetSpan, StringComparison.Ordinal));
```

Comment:
@ahsonkhan why we don't test `span.IndexOf(targetSpan);` as `string` tests, i saw this here and on other tests

Reply:
We could. Feel free to add tests that you feel are missing (or can be consolidated) in a subsequent PR.